### DISCUSSION TITLE
Is If/Else possible in a formula field?

### DISCUSSION THREAD

**Discussion by jpfieber** [mdelobelle/metadatamenu#583] [Q&A]
I'm trying to determine a field value by comparing the date of another field value against today's date. I've tried a number of ways and I'm not getting any results. For example:
`if(current.dateFinished >= (new Date())){"Active"} else {"Closed"}`
the 'dateFinished' property is of type 'Date'.  Are If/Else statements supported, and if so, how could I accomplish my goal?

**Comment by Truncated** [comment]
I've been trying to do the same, albeit with 3+ possible answers so else-if would great as well.
The formula appears to only respond to dv. and current. actions, rather than javascript as a whole, so I've been limited to the dataview functions. However, you can use the dataview stuff in [ternary expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) format to accomplish what you have above:

I think this should work for you, but I can't test it atm:
```
current.dateFinished >= new Date() ? "Active" : "Closed"
```

This was successful for me, as comparison in case the formatting above is suspect.
```
dv.func.contains(current.tags, "Status/Overdue") ? "Red" : "Green"
```
I've nested other Ternary Expressions in the true or false response, but I'm not confident of that working well.

**Comment by jpfieber** [comment]
I modified my workflow so I didn't need this, but thanks for sharing, hope a solution can be found for your need.
